README.md

```diff
@@ -6,29 +6,25 @@
 </h3>
 
 <p align="center">
-	<a href="https://github.com/caarlos0/catppuccin-go/stargazers"><img src="https://img.shields.io/github/stars/caarlos0/catppuccin-go?colorA=363a4f&colorB=b7bdf8&style=for-the-badge"></a>
-	<a href="https://github.com/caarlos0/catppuccin-go/issues"><img src="https://img.shields.io/github/issues/caarlos0/catppuccin-go?colorA=363a4f&colorB=f5a97f&style=for-the-badge"></a>
-	<a href="https://github.com/caarlos0/catppuccin-go/contributors"><img src="https://img.shields.io/github/contributors/caarlos0/catppuccin-go?colorA=363a4f&colorB=a6da95&style=for-the-badge"></a>
+	<a href="https://github.com/catppuccin/go/stargazers"><img src="https://img.shields.io/github/stars/catppuccin/go?colorA=363a4f&colorB=b7bdf8&style=for-the-badge"></a>
+	<a href="https://github.com/catppuccin/go/issues"><img src="https://img.shields.io/github/issues/catppuccin/go?colorA=363a4f&colorB=f5a97f&style=for-the-badge"></a>
+	<a href="https://github.com/catppuccin/go/contributors"><img src="https://img.shields.io/github/contributors/catppuccin/go?colorA=363a4f&colorB=a6da95&style=for-the-badge"></a>
 </p>
 
 ![Catppuccin Variants](https://raw.githubusercontent.com/catppuccin/go/main/_examples/catppuccin.gif)
 
 ## Usage
 
-1. `go get github.com/catppuccin/go`
-2. Add an`import catppuccin "github.com/catppuccin/go"` to your file
-3. Use the flavours as you wish.
+1. Run `go get github.com/catppuccin/go`.
+2. Add `import catppuccin "github.com/catppuccin/go"` to your file.
+3. Refer to the [pkg.go.dev documentation](https://pkg.go.dev/github.com/catppuccin/go) and [`_examples/main.go`](https://github.com/catppuccin/go/blob/main/_examples/main.go) for usage.
```

Comment:
Minor nit/question, but I wonder if this is needed at all. 
Generally a badge for the godoc is where I'd point Go users to expect to find docs/usage.

I'd be happy to do it in a follow-up PR though, no need to update this PR.